src/complex.js

```diff
@@ -161,10 +161,10 @@ Sk.builtin.complex = function (real, imag) {
         ci.real += cr.imag;
     }
 
-    // adjust for negated imaginary literal
-    if (cr.real === 0 && (ci.real < 0 || Sk.builtin.complex._isNegativeZero(ci.real))) {
-        cr.real = -0;
-    }
+    // // adjust for negated imaginary literal
+    // if (cr.real === 0 && (ci.real < 0 || Sk.builtin.complex._isNegativeZero(ci.real))) {
+    //     cr.real = -0;
+    // }
```

Comment:
I commented this out. 
any time the second argument has a negative real part and the first argument is zero then the first argument becomes a negative zero. 

e.g.
```
>>> complex(0, -3)
-0-3i
```
the real 0 becomes a negative zero because the second argument real part (-3) is negative... 
but the first argument 0 should remain the same and not depend on the real part of the second argument at all. 

fixes #1010

Reply:
fixes #1012

Reply:
:+1: please remove the code

Reply:
done with commit abc066f 👍